scripts/curate_basel.py

```diff
@@ -0,0 +1,120 @@
+import os
+import shutil
+import json
+import argparse
+
+
+def get_parameters():
+    parser = argparse.ArgumentParser(description='This script is curating dataset Delbono Wakehealth to BIDS')
```

Comment:
in the future, i suggest to remove the name of the project-- better to _not_ have it than to have something wrong (this is basel, not delbono).

```suggestion
    parser = argparse.ArgumentParser(description='Convert dataset to BIDS format.')
```